pkg/v21/resource/configmapmigration/create.go

```diff
@@ -47,6 +53,67 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 		return microerror.Mask(err)
 	}
 
+	tenantAPIDomain, err := key.APIDomain(clusterConfig)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	tenantG8sClient, err := r.tenant.NewG8sClient(ctx, clusterConfig.ID, tenantAPIDomain)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	tenantK8sClient, err := r.tenant.NewK8sClient(ctx, clusterConfig.ID, tenantAPIDomain)
```

Comment:
I thought we have the clients in the controller context. There is a resource that ensures them. Let's see if we can use that.

Reply:
This is only for the legacy controllers so there is no clients resource. I'll see how much work it is to backport that.

Reply:
@xh3b4sd This is using the controller context now. PTAL